extensions/sns/src/errors.rs

```diff
@@ -0,0 +1 @@
+pub static DFXJSON_NOT_FOUND: &str = "Cannot find dfx configuration file in the current working directory. Did you forget to create one?";
```

Comment:
Super curious why this isn't in https://github.com/dfinity/dfx-extensions/tree/main/extensions-utils/src